user-journey.adoc

```diff
@@ -1,62 +1,91 @@
-= User Journey for Istio/Jaeger
+= User Journey - Installing Istio + Fabric8 Launcher
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
+This document describes how to use the Istio installer to install the following components:
 
-== Flows description
+* Istio platform to connect, manage, and secure microservices.  This Istio installation includes the following components:
+** Elasticsearch
+** Grafana
+** Istio
+** Jaeger
+** Prometheus
+* Fabric8 integrated development platform launcher.
+* Four Istio-specific sample applications (boosters) and tutorials (missions) that let you explore some of the Istio features.
 
-The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+== Platform launcher workflows
 
-. *`Zip flow`:* Selected mission is downloaded from the web site as a zip archive and extracted to the local filesystem.
-. *`Git flow`:* Selected mission is saved to a GitHub repository/account, and an Istio enabled application is created in a local Openshift instance.
+There are two separate workflows that you can use with the Fabric8 development platform launcher:
+
+* *Continuous Delivery workflow* (Recommended) - Use this workflow if you want to automatically create the application components under your local OpenShift instance and also generate a repository for the booster application code under your GitHub account.
+* *Build and run locally workflow* -  Use this workflow if you want to download the booster application files as a zip archive and extract it to your local file system.  Note that with this workflow you have to follow the instructions in the mission README file to deploy the application.
 
 == Prerequisites
-. At least 6 Gb of free RAM, 4 CPUs
-. *Install Java JDK 8* (necessary only for `Zip flow` and boosters using Fabric8 Maven Plugin)
-.. http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[JDK 8]
-.. http://maven.apache.org/download.cgi[Maven 3.5]
-. *Set up Git SCM / GitHub account*
-.. Install https://git-scm.com/downloads[Git SCM client]
-.. Set up a https://github.com[GitHub account]
-.. Configure a GitHub https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[API access token].
-. *Configure certificates for Red Hat Container Image Registry*
-... RHEL - Certificates are already installed. No action is required.
-... Fedora/CentOS/VM - it is required to have this package installed in order to install Red Hat certificate. This allows the docker daemon to access the Red Hat Container Image Registry
+Before you can install Istio and Fabric8, you should already have met the following prerequisites:
```

Comment:
It's not Fabric8, it's the Fabric8 Laucher AFAIK. Also, you previously referredto this as the "platform launcher" --- I think we need to be consistent in what this is called.

Reply:
Will update in both places to make it consistently Fabric8 Launcher.